packages/react-components/react-tree/src/components/TreeItem/useTreeItemContextValues.ts

```diff
@@ -0,0 +1,16 @@
+import * as React from 'react';
+import { TreeItemContextValue } from '../../contexts';
+import type { TreeItemContextValues, TreeItemState } from './TreeItem.types';
+
+export function useTreeItemContextValues_unstable(state: TreeItemState): TreeItemContextValues {
+  const { isActionsVisible } = state;
+
+  const treeItem: TreeItemContextValue = React.useMemo(
+    () => ({
+      isActionsVisible,
+    }),
+    [isActionsVisible],
+  );
```

Comment:
For now, as this context is an object with a single attribute, I see no reason why to use `react-context-selector`. If by any chance this context becomes more complex I'll gladly migrate it